Library/Phalcon/Acl/Adapter/Database.php

```diff
@@ -310,7 +311,7 @@ public function getResources()
      *
      * @return RoleInterface[]
      */
-    public function getRoles()
+    public function getRoles(): RoleInterface
```

Comment:
```suggestion
    public function getRoles(): array
```

Please note **`RoleInterface[]`** !== **`RoleInterface`**